src/components/Navbar/components/NavbarRightList/NavbarMenuItem/NavbarMenuItem.tsx

```diff
@@ -0,0 +1,23 @@
+import Button from "@/components/Button/Button"
+import { theme } from "@/styles/theme"
+import { NavbarMenuProps } from "@/components/Navbar/types"
+import * as S from "./NavbarMenuItem.Styles"
+
+const NavbarMenuItem = ({ children, handleMenuClick }: NavbarMenuProps) => {
+  return (
+    <S.NavbarMenuContainer>
+      <Button
+        $fontColor={theme.colors.white}
+        onClick={() => {
+          // string | undefined 문제..
+          handleMenuClick(children as string)
```

Comment:
현재 NavbarMenuContainer에 children으로 들어오는 값이 전부 string이라 NavbarMenuProps를 이렇게 수정하면 타입 단언을 안 쓸 수 있을 것 같아요!! 🤔
```
export type HandleMenuClickProps = (menu: string) => void

export interface NavbarMenuProps {
  handleMenuClick: HandleMenuClickProps
  $isToggle?: boolean
  children: string
}

```
```suggestion
          handleMenuClick(children)
```
다만 범용적으로 쓰려고 일부러 children?: string으로 하신거라면 잘 모르곘네요....🥲